test/unit/org/apache/cassandra/db/NativeCellTest.java

```diff
@@ -44,7 +48,11 @@
 {
 
     private static final Logger logger = LoggerFactory.getLogger(NativeCellTest.class);
-    private static final NativeAllocator nativeAllocator = new NativePool(Integer.MAX_VALUE, Integer.MAX_VALUE, 1f, null).newAllocator();
+    private static final NativeAllocator nativeAllocator = new NativePool(Integer.MAX_VALUE,
+                                                                          Integer.MAX_VALUE,
+                                                                          1f,
+                                                                          (minOwnershipRatio) -> CompletableFuture.completedFuture(true),
+                                                                          Integer.MAX_VALUE).newAllocator();
     private static final OpOrder.Group group = new OpOrder().start();
```

Comment:
Do we need to close this `Group`? Either way, I suppose we could throw `@SuppressWarnings("resource")` on it and remove what looks like the last compile warning in this class...